doc/model_persistence.rst

```diff
@@ -1,294 +1,307 @@
+
+`ONNX` is a binary serialization of the model. It has been developed to improve
+the usability of the interoperable representation of data models. It aims to
+facilitate the conversion of the data models between different machine learning
+frameworks, and to improve their portability on different computing
+architectures. More details are available from the `ONNX tutorial
+<https://onnx.ai/get-started.html>`__. To convert scikit-learn model to `ONNX`
+`sklearn-onnx <http://onnx.ai/sklearn-onnx/>`__ has been developed. However,
+not all scikit-learn models are supported, and it is limited to the core
+scikit-learn and does not support most third party estimators. One can write a
+custom converter for third party or custom estimators, but the documentation to
+do that is sparse and it might be challenging to do so.
+
+.. _skops_persistence:
+
+`skops.io`
+----------
+
+:mod:`skops.io` avoids using :mod:`pickle` and only loads files which have types
+and references to functions which are trusted either by default or by the user.
+Therefore it provides a more secure format than :mod:`pickle`, :mod:`joblib`,
+and `cloudpickle`_.
+
+
+|details-start|
+**Using skops**
+|details-split|
+
+The API is very similar to :mod:`pickle`, and you can persist your models as
+explained in the `documentation
+<https://skops.readthedocs.io/en/stable/persistence.html>`__ using
+:func:`skops.io.dump` and :func:`skops.io.dumps`::
+
+    import skops.io as sio
+    obj = sio.dump(clf, "filename.skops")
+
+And you can load them back using :func:`skops.io.load` and
+:func:`skops.io.loads`. However, you need to specify the types which are
+trusted by you. You can get existing unknown types in a dumped object / file
+using :func:`skops.io.get_untrusted_types`, and after checking its contents,
+pass it to the load function::
+
+    unknown_types = sio.get_untrusted_types(file="filename.skops")
+    # investigate the contents of unknown_types, and only load if you trust
+    # everything you see.
+    clf = sio.load("filename.skops", trusted=unknown_types)
+
+Please report issues and feature requests related to this format on the `skops
+issue tracker <https://github.com/skops-dev/skops/issues>`__.
+
+|details-end|
+
+.. _pickle_persistence:
+
+`pickle`, `joblib`, and `cloudpickle`
+-------------------------------------
+
+These three modules / packages, use the `pickle` protocol under the hood, but
+come with slight variations:
+
+- :mod:`pickle` is the native Python serialization module. It can serialize and
+  deserialize any Python object, including custom Python classes and objects.
+- :mod:`joblib` is more efficient than `pickle` when working with large machine
+  learning models or large numpy arrays.
+- `cloudpickle`_ can serialize certain objects which cannot be serialized
+  by :mod:`pickle` or :mod:`joblib`, such as user defined functions and lambda
+  functions.
+
+|details-start|
+**Using ``pickle``, ``joblib``, or ``cloudpickle``**
```

Comment:
The rendering will be better.

```suggestion
**Using pickle, joblib, or cloudpickle**
```

Reply:
Sure, but then they have to be un-bolded